settings.gradle.kts

```diff
@@ -1,4 +1,5 @@
 enableFeaturePreview("VERSION_CATALOGS")
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
Ah, I think this answers my question...

Reply:
Yep! Another thing that I'm surprised wasn't a thing before.

Reply:
Yeah, interesting.